float32[] StateAction
---
bool fail
float32 probability